rest/model/task.go

```diff
@@ -173,20 +172,19 @@ func (ad *APIOomTrackerInfo) ToService() (interface{}, error) {
 func (at *APITask) BuildPreviousExecutions(tasks []task.Task, url string) error {
 	at.PreviousExecutions = make([]APITask, len(tasks))
 	for i := range at.PreviousExecutions {
-		if err := at.PreviousExecutions[i].BuildFromService(&tasks[i]); err != nil {
+		if err := at.PreviousExecutions[i].BuildFromArgs(&tasks[i], &APITaskArgs{IncludeProjectIdentifier: true, IncludeAMI: true, IncludeArtifacts: true}); err != nil {
 			return errors.Wrap(err, "error marshalling previous execution")
 		}
 		if err := at.PreviousExecutions[i].BuildFromService(url); err != nil {
 			return errors.Wrap(err, "failed to build logs for previous execution")
 		}
-		if err := at.PreviousExecutions[i].GetArtifacts(); err != nil {
-			return errors.Wrap(err, "failed to fetch artifacts for previous executions")
-		}
 	}
 
 	return nil
 }
 
+// Deprecated: use BuildFromArgs instead
```

Comment:
Can this be a full sentence like "BuildFromArgs should be used instead to add fields that aren't from the task collection" or something.